content/archive/internal/cje30-hw2-jungeunchoi.md

```diff
@@ -0,0 +1,11 @@
+---
+title: "HW2"
+author: "Jungeun Choi"
+date: "2021-03-13T04:58:39Z"
+tags: [Package]
+link: "https://bookdown.org/cje30/hw2_jungeunchoi/"
+length_weight: "5.2%"
+pinned: false
+---
+
+This is a minimal example of using the bookdown package to write a book. The output format for this example is bookdown::gitbook. [...] 최정은, 의예과, 2060070  ...
```

Comment:
Trying to add batch commit using suggestion to remove all these examples. 

They seems like assignement.

```suggestion
```